include/vcpkg/base/messages.h

```diff
@@ -1220,6 +1232,10 @@ namespace vcpkg
                     "{value} is a localized message name like LocalizedMessageMustNotEndWithNewline",
                     "The message named {value} ends with a newline which should be added by formatting "
                     "rather than by localization.");
+    DECLARE_MESSAGE(ManifestConflict,
+                    (msg::path),
+                    "",
+                    "Found both a manifest and CONTROL files in port {path}; please rename one or the other");
```

Comment:
```suggestion
                    "Found both a manifest and CONTROL files in port {path} ; please rename one or the other");
```

Same path issue here.

Reply:
This is what the old code did and I don't think the extra space is an improvement; I know I argued against quoting all paths but maybe this is an argument that we should sometimes?